src/v2/BaseLoginRouter.js

```diff
@@ -153,7 +153,25 @@ export default Router.extend({
   },
 
   handleIdxResponseFailure (error = {}) {
-    // 1. loosely check whether is IDX error response
+    // special case: `useInteractionCodeFlow` is true but the Org is not using OIE
+    const featureMsg = 'The requested feature is not enabled in this environment.';
```

Comment:
the error messages needs to be translated..

Reply:
This is the error that comes from the backend